src/bp/ui-studio/src/web/views/OneFlow/sidePanel/form/SaySomethingForm.tsx

```diff
@@ -183,6 +183,7 @@ const SaySomethingForm: FC<Props> = props => {
           schema={currentContentType?.schema.json}
           uiSchema={currentContentType?.schema.ui}
           formData={currentFlowNode?.content?.formData}
+          customKey={currentFlowNode?.name}
```

Comment:
i find it a bit weird to call this custom key . Can't we simply pass it as  nodeName ? or nodeId ?

Reply:
Or is it possible to simply not pass the key down to children and simply hash the formData and use this as key ? re-render only when the key changes (aka form data changes) or that won't work if form data is empty for 2 nodes ?

Reply:
I called it customKey since anything could be passed down if needed, not necessarily only a node name.

I tried based on formData and it re-renders every single time you type a letter, it causes a noticeable performance issue.